doc/ref/obsolete.xml

```diff
@@ -192,9 +192,7 @@ your specific use case.
 is an info class to display warnings when an obsolete variable is used.
 By default, these warnings are switched off since the info level for 
 this class is 0. Setting it to 1 will trigger warnings if &GAP; will 
-detect that an obsolete variable is used at runtime (this detection is
-possible, however, only for obsolete variables declared using 
-<C>DeclareObsoleteSynonym</C>). 
+detect that an obsolete variable is used at runtime.
```

Comment:
Or perhaps change it to `obsolete identifier`, then explain that we cannot resp.  "do not warn about obsolete identifiers for global variables, only for global functions, operations, attributes and properties".